roles/user/tasks/accounts.yml

```diff
@@ -0,0 +1,61 @@
+---
+- name: Add groups
+  become: true
+  ansible.builtin.group:
+    name: "{{ item['name'] }}"
+    state: present
+  with_items:
+    - "{{ user_list }}"
```

Comment:
maybe I'm still a bit brain afk, but shouldn't this role also delete empty user groups when the last user get's removed?

Reply:
well the groups we create are user-specific, there shouldn't be anyone else in them. not sure if that holds true for pre-existing accounts that are to be deleted. adding logic to check this might be possible, but would make the role more complicated, not sure if that is worth the effort

Reply:
well I don't know the scope, but we delete users in this role. and if these are single user groups it's quite logical to also remove the groups, when the single user in it is removed, no?

Reply:
I have tested this, and the ansible user module does in fact also delete the primary group for a user when the `state=absent` action is performed. Since I haven't found any mention of that in the docs though, do we still want to add an explicit "Remove groups" task just to be on the safe side, even if that task may never be executed?

Reply:
I would prefer an explicit remove groups task.